pkg/machines/components/libvirtSlate.jsx

```diff
@@ -0,0 +1,79 @@
+/*jshint esversion: 6 */
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+import React from "react";
+import cockpit from 'cockpit';
+import { mouseClick } from "../helpers.es6";
+import { startLibvirt } from "../actions.es6";
+
+const _ = cockpit.gettext;
+
+const LibvirtSlate = ({ libvirtState, dispatch }) => {
+    const goToLink = () => cockpit.jump("/system/services#/libvirtd.service");
+    const start = () => dispatch(startLibvirt());
+
+    let message;
+    let icon;
+    let detail;
+    let action;
+
+    switch (libvirtState.status) {
+        case 'active':
+            message = _("Virtualization service is available");
+            icon = (<span className="pficon-ok"/>);
+            break;
+        case 'unknown':
+            message = _("Connecting to virtualization service");
```

Comment:
This should also be headline text, so:

```jsx
message = _("Connecting to Virtualization Service");
```

It _might_ also be better with "the" in the string, as it's a virtualization service (and not something _named_ Virtualization Service as a proper noun), like so the following is even a bit more ideal:

```jsx
message = _("Connecting to the Virtualization Service");
```